packages/aws-cdk-lib/aws-globalaccelerator/lib/accelerator.ts

```diff
@@ -92,10 +164,16 @@ export class Accelerator extends cdk.Resource implements IAccelerator {
     const resource = new ga.CfnAccelerator(this, 'Resource', {
       enabled: props.enabled ?? true,
       name,
+      ipAddresses: props.ipAddresses ?? undefined,
+      // ipAddressType is undefined as a default here for backwards compatibility,
+      // but the service selects "IPV4".
+      ipAddressType: props.ipAddressType ?? undefined,
```

Comment:
I don't think the null coalescing operators are necessary since they are originally union types with undefined.

```suggestion
      ipAddresses: props.ipAddresses,
      ipAddressType: props.ipAddressType,
```